tests/test_submission.py

```diff
@@ -169,14 +169,140 @@ def test_get_metadata_submissions_report_as_admin(
         author_orcid=other_user.orcid,
         created=now + timedelta(seconds=1),
         date_last_modified=datetime.utcnow(),
-        is_test_submission=True,
         metadata_submission={
```

Comment:
I think this test data can be distilled to only the parts that are required by the submission schema and that impact the functionality of this test. I suggest adding a `TODO` comment about it.

```suggestion
        # TODO: Omit some optional fields in order to simplify the test data.
        # See: `class MetadataSubmissionRecordCreate` in `schema_submission.py`
        # See: https://microbiomedata.github.io/submission-schema/SampleData/
        metadata_submission={
```